packages/courDeCassation/src/connector/mapper/extractors/extractRoute/extractRoute.ts

```diff
@@ -27,6 +28,10 @@ function extractRoute(
   switch (source) {
     case 'jurinet':
       try {
+        logger.error({
+          operationName: 'extractRouteForJurinet',
+          msg: `${routeInfos}`,
```

Comment:
`"extractRouteForJurinet","msg":"[object Object]"` ça log ça ducoup :/

`docker container exec -it label_backend_1 sh -c "cd packages/courDeCassation; sh scripts/runLocalScript.sh ./dist/scripts/updateRoute.js"` pour tester en local :wink: 

Et enfait mieux vaut ne pas logger les `routeInfos`, il y a les parties dedans (données sensibles)